chains/solana/contracts/programs/ccip-offramp/src/instructions/interfaces.rs

```diff
@@ -95,7 +99,7 @@ pub trait Admin {
     fn set_ocr_config(
         &self,
         ctx: Context<SetOcrConfig>,
-        plugin_type: u8, // OcrPluginType, u8 used because anchor tests did not work with an enum
+        plugin_type: OcrPluginType,
```

Comment:
Is this a breaking change?  If yes, then we need to inform tooling team to update their interactions as well.